src/scenes/AcceleratorRouter/Deliverables/index.tsx

```diff
@@ -15,7 +18,33 @@ const DeliverablesRouter = () => {
         element={
           <ProjectProvider>
             <ParticipantProvider>
-              <DeliverableViewWrapper />
+              <ParticipantProjectSpeciesProvider>
```

Comment:
Can you move this lower in the tree? Ideally this provider only wraps species related views. I think the "[currentDeliverable](https://github.com/terraware/terraware-web/pull/2596/files#diff-08120ee089f7fa1ecb53e3f3942ddf0fbccd748b043a9f92ee4ae98c341e90e0R22)" logic can be moved to the `ParticipantProvider`. This way you can do `const { currentDeliverable } = useParticipantData()` within the `ParticipantProjectSpeciesProvider`.

In order to do this, you will need to move the "get current deliverable from params" [code](https://github.com/terraware/terraware-web/pull/2596/files#diff-d1ed142f473884830fdf70b706960a57f9f9969f94e42deb200a8a081fcf8389R37) within the `DeliverableViewWrapper` into the `ParticipantProvider`. 

That provider currently has information about active module, phase, etc... so I think it makes sense to also add active deliverable there too.

Another option, that might be better but slightly more work, is to create a new `DeliverableProvider` that manages the current deliverable.

In the router:
```
<ProjectProvider>
  <ParticipantProvider>
    <DeliverableProvider>
      <DeliverableViewWrapper />
    </DeliverableProvider>
  </ParticipantProvider>
</ProjectProvider>
```

In the wrapper:
```
        return (
          <ParticipantProjectSpeciesProvider>
            <SpeciesDeliverableView deliverable={deliverable} />
          </ParticipantProjectSpeciesProvider>
        );
```

Reply:
Sure, I will create DeliverableProvider, sounds better

Reply:
I tried a lot of different ways to have the ParticipantProjectSpeciesProvider only before the <SpeciesDeliverableView> but then when I tried to get the ParticipantProjectSpecies in the others view, it was always undefined. Maybe I'm doing something wrong. 

I added new routers to make it easier to follow

Reply:
In the new views I created for View species (src/scenes/AcceleratorRouter/Species/index.tsx) and edit species (src/scenes/AcceleratorRouter/Species/SpeciesEditView.tsx)

Reply:
Can you take a look at this branch and see if it works for you? https://github.com/terraware/terraware-web/pull/2623/files

Reply:
Oh nevermind, I think I see the issue... will push up another commit here in a bit

Reply:
No, it doesn't work. It still doesn't get the participantPojectSpecies correctly, because it is set in the Table (that it is inside the SpeciesDeliverableView, inside the delierableViewWrapper). So we need the provider there too